programs/sbf/Cargo.lock

```diff
@@ -4790,9 +4802,9 @@ dependencies = [
 
 [[package]]
 name = "serde_json"
-version = "1.0.138"
+version = "1.0.140"
```

Comment:
This also got bumped one additional patch version compared to the main workspace (1.0.140 here vs 1.0.139 there), but should be ok